src/components/DisplayContainer/DisplayContainer.scss

```diff
@@ -1,6 +1,23 @@
-.display-container {
+@import '../../variables';
+
+
+.card-container {
   display: flex;
   flex-wrap: wrap;
   justify-content: space-around;
   padding: 10px;
-}
\ No newline at end of file
+}
+
+.scroll-container {
+  font-size: 1.5em;
+  line-height: 1.4em;
```

Comment:
The line-height increase make this, and the character cards much more readable.  Thanks for adding that.